src/examples/DrawingTool/DrawingTool.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import {
+  AzureMap,
+  AzureMapDataSourceProvider,
+  AzureMapDrawingManagerProvider,
+  AzureMapFeature,
+  AzureMapLayerProvider,
+  AzureMapsProvider,
+  IAzureMapOptions,
+} from 'react-azure-maps';
+import { AuthenticationType } from 'azure-maps-control';
+import { key } from '../../key';
+import { control, drawing } from 'azure-maps-drawing-tools';
+import Description from '../../Layout/Description';
+import { Button } from '@material-ui/core';
+import { wrapperStyles } from '../Options/ChangeOptionsWrapper';
+
+const option: IAzureMapOptions = {
+  authOptions: {
+    authType: AuthenticationType.subscriptionKey,
+    subscriptionKey: key,
+  },
+};
+
+const drawingOptions = {
+  toolbar: new control.DrawingToolbar({
+    position: 'top-right',
+    style: 'dark',
+  }),
+};
+
+const mode = {
+  mode: drawing.DrawingMode.drawPolygon,
+};
+
+const DrawingTool: React.FC = () => {
+  const [options, setOptions] = useState<any>(drawingOptions);
+  return (
+    <div>
+      <Description>Change options in child</Description>
+      <div style={wrapperStyles.buttonContainer}>
+        <Button size="small" variant="contained" color="secondary" onClick={() => setOptions(mode)}>
+          Change Bounds
+        </Button>
+      </div>
+      <AzureMapsProvider>
+        <div style={{ height: '300px' }}>
+          <AzureMap options={option}>
+            <AzureMapDrawingManagerProvider options={options}>
+              <AzureMapDataSourceProvider
```

Comment:
@psrednicki: do you want this datasource to wrap the datasource that drawing manager creates right? 
Since this datasource has no association with drawing manager and thus the feature defined in it won't be editable as https://github.com/WiredSolutions/react-azure-maps/issues/70#issuecomment-718349117 mentioned here?